src/frontend/src/handler/alter_source_column.rs

```diff
@@ -144,7 +144,7 @@ pub fn alter_definition_add_column(definition: &str, column: ColumnDef) -> Resul
         _ => unreachable!(),
     }
 
-    Ok(stmt.to_string())
+    Ok(stmt.to_unredacted_string())
```

Comment:
May I ask how you find all references of `to_string` call on `Statement`? That is to say, how could we avoid showing an unredacted string where we should redact it in the future or vice versa?

Reply:
> how you find all references of to_string call on Statement?

Most of them are found by test failure 🥵 

> how could we avoid showing an unredacted string where we should redact it 

to_string or display are redacted by default.

> or vice versa

Caller is responsible for explicitly using `to_unredacted_string`. Similar to [crdb's behavior](https://github.com/risingwavelabs/risingwave/issues/12673#issuecomment-1863903024): "everything gets redacted except for those bits which we know are safe"